solar/solar/interfaces/orm.py

```diff
@@ -477,6 +483,40 @@ def add_input(self, name, schema, value):
         self.inputs.add(input)
 
 
+class DBEvent(DBObject):
+
+    __metaclass__ = DBObjectMeta
+
+    _collection = base.BaseGraphDB.COLLECTIONS.events
+
+    id = db_field(is_primary=True)
+    parent = db_field(schema='str!')
+    parent_action = db_field(schema='str!')
+    etype = db_field('str!')
+    state = db_field('str')
+    child = db_field('str')
+    child_action = db_field('str')
+
+    @classmethod
+    def load_list(cls, parent):
+        rs = db.all(collection=cls._collection.name + ':' + parent)
```

Comment:
Why are you making keys that don't exist. The `_collection` parameter is part of enum and should stay this way. If you want relation to resource, use the `DBRelatedField`.